### DISCUSSION TITLE
syntax for Edit Me button in a Jekyll site

### DISCUSSION THREAD

**Discussion by Texterity** [community/community#22330] [Other Feature Feedback, Questions, & Ideas]
<p><img src="https://user-images.githubusercontent.com/38268019/181070210-512c87e7-f8fd-41a7-868f-5f2ee8dfbdd1.jpeg" alt="EditMe.jpg" data-base62-sha1="yxxcvBgosXeRDAbsEabRs91MGNI" width="118" height="39"></p>
<p>I’m trying to update the url for the Edit Me button (above) on a Jekyll site I’m building, by entering the basepath URL in the config.yml file.</p>
<p>I tried using the full URL:</p>
<p><a href="https://github.com/%5Bprofilename%5D/%5Breponame%5D.git" rel="nofollow noopener">https://github.com/[profilename]/[reponame].git</a></p>
<p>And all kinds of combinations of /[profilename]/[reponame]</p>
<p>I have published the site using a cname record to &lt;<a href="http://docs.mysitename.com" rel="nofollow noopener">docs.mysitename.com</a>&gt;</p>
<p>But every combination produces a 404 error. Any tips or guidance appreciated.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/texterity">@texterity</a>,</p>
<p>Is your repository public? If so, could you share a link to it so that we can check out the code you’re working with? I think that will help for troubleshooting this.</p>
<p>Thanks!</p>

**Comment by Texterity** [comment]
<p>Actually no - which might be part of the problem, now you mention it! I will create a public version and try and again, and if that doesn’t solve the problem, then I will post it. </p>

**Comment by Texterity** [comment]
<p>OK I’ve made that repo public at <a href="http://docs.texterity.com.au" rel="nofollow noopener">http://docs.texterity.com.au</a> (It’s a copy of the Jekyll Documentation template from the idratherbewriting blog.) There are currently two branches, Master and Review. I had the idea that I could point the Edit Me link to the Review branch. But the desired effect is that, anonymous users (i.e. not signed in to Github) can propose edits. </p>

**Comment by Texterity** [comment]
<p>Incidentally in the idratherbewriting template the basepath url is</p>
<p>tomjoht/documentation-theme-jekyll/blob/gh-pages/</p>
<p>Is the term ‘blob’ required for this expression?</p>

**Comment by Texterity** [comment, marked as answer]
<p>Answered my own question! Open the repo and branch that you want to be the destination for the edit path, then copy the URL between /yourid and the file name. That becomes the base path for the config.yml.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/texterity">@texterity</a>, Thank you for coming back and sharing what worked for you!</p>
